lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -140,6 +144,34 @@ function findDifference(path, actual, expected) {
       continue;
     }
 
+    if (key === '_excludes') {
+      // Re-use state from `_includes` check, if there was one.
+      /** @type {any[]} */
+      const arrToCheckAgainst = inclExclCopy || actual;
+
+      if (!Array.isArray(expectedValue)) throw new Error('Array subset must be array');
+      if (!Array.isArray(actual)) continue;
+
+      const unexpectedValues = expectedValue;
```

Comment:
Lol @ expected -> unexpected. Could we make this `expectedExclusions` or something?